devel/gettext/Portfile

```diff
@@ -6,9 +6,7 @@ PortGroup               muniversal 1.0
 PortGroup               clang_dependency 1.0
 
 name                    gettext
-epoch                   2
```

Comment:
A port's epoch can never be decreased so `epoch 2` must stay, however ideally it should be in the `if {${subport} eq ${name}}` section so that it does not apply to the new subports being added here.

Reply:
I added the epoch to the `if {${subport} eq ${name}}` section.